---
title: "'Throw' ifadesi işleneni, 'Catch' deyimi dışında veya 'Finally' deyimi içinde atlayamazsınız"
ms.date: 07/20/2015
f1_keywords:
- vbc30666
- bc30666
helpviewer_keywords:
- BC30666
ms.assetid: a208a6ea-0e36-4bf1-8984-4de1a0e38a2a
ms.openlocfilehash: 17185ec328cecc29560b28e878682005e2a8a9e8
ms.sourcegitcommit: 0888d7b24f475c346a3f444de8d83ec1ca7cd234
ms.translationtype: MT
ms.contentlocale: tr-TR
ms.lasthandoff: 12/22/2018
ms.locfileid: "53768123"
---
# <a name="throw-statement-cannot-omit-operand-outside-a-catch-statement-or-inside-a-finally-statement"></a>'Throw' ifadesi işleneni, 'Catch' deyimi dışında veya 'Finally' deyimi içinde atlayamazsınız
`Throw` deyimleri dışında `Catch` deyimi bir özel durum nesnesi adı sağlamanız gerekir.  
  
 **Hata Kimliği:** BC30666  
  
## <a name="to-correct-this-error"></a>Bu hatayı düzeltmek için  
  
1.  Türetilen bir özel durum nesnesi adını `System.Exception`.  
  
2.  Kodunuzu yeniden yapılandırmayı böylece `Throw` deyimdir içinde bir `Catch` blok.  
  
## <a name="see-also"></a>Ayrıca Bkz.  
 [Throw Deyimi](../../visual-basic/language-reference/statements/throw-statement.md)  
 [Try...Catch...Finally Deyimi](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 [Visual Basic'te özel durum sınıfı](https://msdn.microsoft.com/library/9aac396f-34ca-4afb-8e6c-e523cb690ba9)  
 [Özel durum ve Visual Basic'te işleme hatası](https://msdn.microsoft.com/library/3e351e73-cf23-40ab-8b60-05794160529e)
